publishing.md

```diff
@@ -53,8 +53,12 @@ Please make sure to update the destination of the source code link if you use a
 </span>
 ```
 
-## Changes
+## Distributing changes
 
-If you modify or extend OpenFisca, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
+If you modify or extend OpenFisca, and that you distribute those changes, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, country package, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
```

Comment:
I read this as explicitly saying that it is okay to derive a better model from the shared one and create reports from it without sharing the improvements, as I'm not “distributing the changes” 😕

Reply:
It is the way I understand the AGPL-3.0 license, and I'm not necessarily happy about that. If you have a reputable source that says otherwise, I'm more than willing to change it. For now, my understanding is that share-back triggers once _use_, _modification_, **and** _distribution_.

Reply:
(I guess I owe you a source as well)

Reply:
For the first one:

> Making copies of the software for yourself is the main form of propagation that is not conveying. You might do this to install the software on multiple computers, or to make backups.

https://www.gnu.org/licenses/gpl-faq.html#PropagationNotConveying

Regarding the latter :

> If I only make copies of a GPL-covered program and run them, without distributing or conveying them to others, what does the license require of me?
> Nothing. The GPL does not place any conditions on this activity.

https://www.gnu.org/licenses/gpl-faq.html#NoDistributionRequirements

Apparently, as it is not considered conveying (you're just sharing the output of your changes, not the changes themselves).